app/src/main/java/org/mozilla/focus/widget/BrowserNavigationOverlay.kt

```diff
@@ -147,6 +147,18 @@ class BrowserNavigationOverlay @JvmOverloads constructor(
 
         pinButton.isChecked = navigationStateProvider?.isURLPinned() ?: false
 
+        // Prevent the focus from looping to the bottom row when reaching the last
+        // focusable element in the top row
+        navButtonReload.nextFocusLeftId = when {
```

Comment:
fwiw, this might fix itself if we used a different layout pattern (instead of the nested LinearLayouts) but we agreed it wasn't worth fixin atm.